packages/zod/src/v4/core/tests/locales/he.test.ts

```diff
@@ -0,0 +1,370 @@
+import { beforeEach, describe, expect, test } from "vitest";
+import { z } from "../../../../index.js";
+import he from "../../../locales/he.js";
+
+describe("Hebrew localization", () => {
+  beforeEach(() => {
+    z.config(he());
+  });
+
+  describe("too_small errors with definite article and gendered verbs", () => {
+    test("string type (feminine - צריכה)", () => {
+      const schema = z.string().min(3);
+      const result = schema.safeParse("ab");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe("קטן מדי: המחרוזת צריכה להיות >=3 אותיות");
```

Comment:
I would personally go with:

```suggestion
        expect(result.error.issues[0].message).toBe("קצר מדי: המחרוזת צריכה להכיל 3 תווים או יותר");
```

just because this is closer to "real" hebrew

(also for arrays, numbers, etc.)

wdyt?

Reply:
amazing. added